torch/autograd/function.py

```diff
@@ -347,6 +348,8 @@ def unflatten_helper(input, proto):
 
 
 class NestedIOFunction(Function):
+    # The 'type: ignore' statements are needed here because these functions are declared as '@staticmethod' in the
+    # superclass (Function) but are instance methods here, which mypy reports as incomptabile.
```

Comment:
Hmm, mypy 0.770 seems happy without the ignores. What's the error you're seeing?

Reply:
Also, typo in `incompatible`

Reply:
I cargo culted this from the old pyi file. I'll go ahead and remove it.